src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -1,12 +1,46 @@
 package frc.robot.subsystems.hood;
 
 import edu.wpi.first.math.geometry.Translation2d;
-import edu.wpi.first.units.Dimensionless;
-import edu.wpi.first.units.Measure;
-import edu.wpi.first.units.Units;
+import edu.wpi.first.units.*;
+import frc.robot.Constants;
+import lib.webconstants.LoggedTunableNumber;
 
 public class HoodConstants {
     public static Translation2d MECHANISM_2D_POSE = new Translation2d(1, 1);
-    public static double HOOD_LENGTH = 0.4; // m
+    public static Measure<Distance> HOOD_LENGTH = Units.Meters.of(0.4);
     public static Measure<Dimensionless> POSITION_TOLERANCE = Units.Percent.of(5);
+    public static double MAX_VELOCITY = 1; // RPS
+    public static double MAX_ACCELERATION = 4; // RPS squared
+    public static final double GEAR_RATIO = 1.0;
+    public static final double MOMENT_OF_INERTIA = 0.0003;
+    public static final double HOOD_X = -0.27;
+    public static final double HOOD_Y = 0.2385;
+    public static final Translation2d ROOT_POSITION = new Translation2d(HOOD_X, HOOD_Y);
```

Comment:
Unless `HOOD_X` and `HOOD_Y` are used somewhere else than `ROOT_POSITION`, you can just write the values directly in `ROOT_POSITION`'s decleration.

Reply:
a2e31af